.github/workflows/reusable-end-to-end-testing.yml

```diff
@@ -110,5 +124,12 @@ jobs:
           VMCLARITY_E2E_SCANNER_IMAGE: ${{ format('{0}:{1}', needs.images.outputs.cli-image, inputs.image_tag) }}
           VMCLARITY_E2E_CR_DISCOVERY_SERVER_IMAGE: ${{ format('{0}:{1}', needs.images.outputs.cr-discovery-server-image, inputs.image_tag) }}
           VMCLARITY_E2E_SCANNER_PLUGIN_KICS: ${{ format('{0}:{1}', needs.images.outputs.plugin-kics-image, inputs.image_tag) }}
+          VMCLARITY_E2E_PLATFORM: ${{ inputs.platform }}
         run: |
```

Comment:
I know I am a bit late but it seems like duplication to me, we don't need two separate make targets if the only difference between them is the value of the `VMCLARITY_E2E_PLATFORM`, which we already set here in the CI, what do you think?

Reply:
but if you mean here, we need to reuse this workflow for each one in order to run the two environments in parallel

Reply:
What I propose is to have only one `make e2e` make target because the difference between running it with docker vs kubernetes provider is just the value of the `VMCLARITY_E2E_PLATFORM` env var. So the `run` command here in the github action can also remain `make e2e` since this env is being set for this step.

Reply:
Alternatively, in the case we stick to having two different make targets for the convenience of running both docker and kubernetes at once with `make e2e`, then setting the `VMCLARITY_E2E_PLATFORM` env here in this GHA step is redundant since the make targets will start the test with the correct platform anyways :)